docs/sphinx/reference-frontend-api.rst

```diff
@@ -949,3 +957,22 @@ Functions
                       This uses the undo action from the first and the redo action from the last action.
 
    .. versionadded:: 29.1
+
+---------------------------------------
+
+.. function:: void obs_frontend_multitrack_video_register(const char *name, multitrack_video_start_cb start_video, multitrack_video_stop_cb stop_video, void *param)
```

Comment:
As much as we flip flop in the code base, the frontend API is using the `private_data` name everywhere else, so it should probably be that.

It may be a good idea to prefix the callback types with `obs_frontend_`, but that would also make them quite long. Probably good for consistency, however I am open to discussion from others.

Reply:
changed both in https://github.com/obsproject/obs-studio/pull/10635#commits-pushed-f65594f

length of the callback type seems fine, though it is on the long side; I don't have a good alternative suggestion at this point though

Reply:
@tt2468 Looks like this feedback was addressed.